.circleci/config.yml

```diff
@@ -1308,6 +1308,25 @@ executors:
     working_directory: /repo
 
 workflows:
+  GCP-Personal-Dev:
+    jobs:
+      - build-platform-and-services:
+          filters:
+            branches:
+              only:
+                - developers-branch-to-run-tests
```

Comment:
Shall this be changed to `master` for merging to master?

Reply:
This should not be run for each commit on master. It is a workflow for developers to easily automate their runs using their personal config. Without this workflow, we need to modify the Daily or Weekly nightly jsons to run tests.

Reply:
The intention is legit, but the implementation is not CircleCi way to do so. IMO, if developers want to run their workflows, either complete new one, or one of existing daily or weekly workflow, they should create a branch on both `hedera-services` repo and `regression` repo. And all the branches won't get interfered with `master` branch at all. Adding these workflows may confuse our understanding of CircleCi workflow, and also make the config.yml more complicated.

Reply:
@ljianghedera I see. Sure I removed the workflow.